components/FlexibleForms/RepeaterField.test.tsx

```diff
@@ -0,0 +1,123 @@
+import RepeaterField from './RepeaterField';
+import { Formik, Form } from 'formik';
+import { render, screen } from '@testing-library/react';
+
+const mockSubmit = jest.fn();
+
+describe('RepeaterField', () => {
```

Comment:
I think we are missing a couple of tests regarding the functionality for adding/removing fields.

Reply:
that feels like testing formik internals to me, no? there's no state or anything involved, it all comes from formik's hooks

Reply:
changed my mind - can't hurt.